webapp/src/app/app.module.ts

```diff
@@ -48,7 +48,7 @@ import { UserData } from "./user-data.service";
 // This is empty as the UserData is now initialized in the
 // app.component, so that the success can be logged.
 export function loadUserDataConfig(d: UserData) {
-    return () => {};
+    return () => {const a = 2; };
```

Comment:
Why?

Reply:
cannot be empty else tslint complains...

Reply:
Does removing `loadUserDataConfig` and the block using it in `providers` (so that `providers: [UserData]`) works?

Reply:
no, because `useFactory` is needed in a provider and must return a function.